### DISCUSSION TITLE
Creating images with the same two faces repeatedly

### DISCUSSION THREAD

**Discussion by RebelAries** [lllyasviel/Fooocus#2201] [Q&A]
First, let me apologize for my lack of tech skills.

As for my inquiry, there are two things I'm trying to accomplish:

1) I'd like to create multiple images of the same two people (they can be AI generated faces instead of based on real people, I just want the same two faces to repeat when I set fooocus to produce multiple images). 

Right now my instructions for fooocus are:
"a full body photograph of a thirty year old couple wearing athletic clothing standing side by side outdoors" and for the negative prompt I put "hand holding, belts, and shirtless" because previously fooocus was making the male figure shirtless and wearing a belt despite wearing shorts with an elastic band. Performance is prioritizing quality, the aspect ratio is set at 1152x896 (9:7), and the image number is 32. 

2) I'd also like to find a way to have some sort of input image of two faces so that, when given the proper instructions, fooocus can produce images of my girlfriend and I in ancient historical locations or alien planets, etc. 

...

Once again, I apologize that I had trouble finding these answers otherwise (I did a search for similar discussions) and for having a lack of tech skills.

**Comment by mashb1t** [comment]
Please check out https://github.com/lllyasviel/Fooocus/discussions/2036#discussioncomment-8373209 and the whole thread :)

**Comment by patrickchamp** [comment]
Try adding this to your prompt “the man on the left has x hair, and the other has x hair”
Specifying the same type of characteristic for both will give you different people. Even if it’s just one simple characteristic. I find that mentioning the word man twice doesn’t work as well.
Also, you’ll have more success using dimension 1:1 even without specifying their appearance. Certain models are better/worse at generating separate appearances so try a different model.

**Reply by sandrapisano** [reply]
Good morning, I also have the same problem. I can't make a couple picture with two separate portraits of a man and a woman that I would like in the same photo. How can I have these two characters in the same photo?
